### DISCUSSION TITLE
How do I download these models？

### DISCUSSION THREAD

**Discussion by 7758521momo** [iTwin/itwinjs-core#3636] [Q&A] [answered]
https://developer.bentley.com/tutorials/create-test-imodel-sample/#house-sample
Generate snapshots locally for learning，How do I download these models，Thanks!

**Comment by calebmshafer** [comment, marked as answer]
@7758521momo what type of application are you building to try and use the snapshots locally? If you're writing a web application based on the iTwin Viewer, it will be more difficult to do that locally as it requires more setup when not using the hosted version of the iModel.

Regardless of the answer to the above, here are the steps to get a copy for one of the sample iModels:

1. Follow that tutorial in the link provided to create your own copy of the sample  iModel

2. Navigate to the [My iModels](https://developer.bentley.com/my-imodels/) page

3. Click the "View in CONNECT" button as shown here,

![Image 5-17-22 at 11 14 PM](https://user-images.githubusercontent.com/31107829/168949942-85659419-ae75-4799-877e-eca13952823e.jpg)

4. Once you're in the iModelHub page in CONNECT, go to the "View all versions",

![Image 5-17-22 at 11 17 PM](https://user-images.githubusercontent.com/31107829/168950159-ccbb51c3-a4ad-4261-a7f3-a895db117897.jpg)

5. Last step, choose the latest changeset and click the download button,

![Image 5-17-22 at 11 18 PM](https://user-images.githubusercontent.com/31107829/168950332-3030a1c9-4747-4e4c-9060-395cd3d9ba06.jpg)

**Reply by 7758521momo** [reply]
just this thanks!

**Reply by oussamabonnor1** [reply]
> @7758521momo what type of application are you building to try and use the snapshots locally? If you're writing a web application based on the iTwin Viewer, it will be more difficult to do that locally as it requires more setup when not using the hosted version of the iModel.
> 
> Regardless of the answer to the above, here are the steps to get a copy for one of the sample iModels:
> 
> 1. Follow that tutorial in the link provided to create your own copy of the sample  iModel
> 2. Navigate to the [My iModels](https://developer.bentley.com/my-imodels/) page
> 3. Click the "View in CONNECT" button as shown here,
> 
> ![Image 5-17-22 at 11 14 PM](https://user-images.githubusercontent.com/31107829/168949942-85659419-ae75-4799-877e-eca13952823e.jpg)
> 
> 4. Once you're in the iModelHub page in CONNECT, go to the "View all versions",
> 
> ![Image 5-17-22 at 11 17 PM](https://user-images.githubusercontent.com/31107829/168950159-ccbb51c3-a4ad-4261-a7f3-a895db117897.jpg)
> 
> 5. Last step, choose the latest changeset and click the download button,
> 
> ![Image 5-17-22 at 11 18 PM](https://user-images.githubusercontent.com/31107829/168950332-3030a1c9-4747-4e4c-9060-395cd3d9ba06.jpg)

@calebmshafer This option is no longer available, all I found was the IFC export. Has it been removed or did I do something wrong?

**Comment by JohnMajerle** [comment]
I don't see the Download feature either and I would like to use it. Is it available in the UI elsewhere?
